flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/proxy/KinesisProxySyncV2.java

```diff
@@ -114,6 +104,7 @@ public DescribeStreamConsumerResponse describeStreamConsumer(
                         .streamARN(streamArn)
                         .consumerName(consumerName)
                         .build();
+        LOG.warn("describeStreamConsumer with arn: {}, consumerName: {}", streamArn, consumerName);
```

Comment:
Why is this a warning? (did you leave dev debug logs in?)

Reply:
Yes, my apology for missing this in post-cleanup, doing a round of check again to remove them